### DISCUSSION TITLE
Welcome to captcha-solver-selenium-python-examples Discussions!

### DISCUSSION THREAD

**Discussion by dzmitry-duboyski** [2captcha/captcha-solver-selenium-python-examples#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by muh33z** [comment]
Does anyone know how to bypass the captcha on Amazon, I've been trying to scrape that site for a while now.

**Reply by dzmitry-duboyski** [reply]
What type of captcha is located on your page ?
If you have Amazom WAF captcha, you can try to use https://2captcha.com/api-docs/amazon-aws-waf-captcha method.
